### PR TITLE
Fix #226 (cert-gen script should be part of adb-utils package)

### LINKED ISSUE #226: Remove cert generation script from kickstart file and make it part of adb-utils package
We are in process of making our kickstart file cleaner and we want script part of it should move to adb-utils. Currently cert generation script is part of this repo and also in kickstart file which we have to remove.

### PR DESCRIPTION
**Pull request by praveenkumar**
Refer #226

### CODE REVIEW

File: build_tools/kickstarts/centos-7-adb-vagrant.ks [hunk 1]
```diff
@@ -128,67 +128,6 @@ echo "ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4
...
-ICRET0NLRVJfQ0VSVF9QQVRIL2NhLnBlbSAkRE9DS0VSX0NFUlRfUEFUSC9zZXJ2ZXItY2VydC5w
-ZW0gJERPQ0tFUl9DRVJUX1BBVEgvc2VydmVyLWtleS5wZW0KCiMjIFJlbW92ZSByZW1haW5pbmcg
-ZmlsZXMKY2QKcm0gLXJmICRkaXIKCiMgRW5kIG9mIGNlcnRzIHBhaXIgZ2VuZXJhdGlvbiBzdGVw
-cyBmb3IgVExTIGVuYWJsZWQgZG9ja2VyIGRhZW1vbgo=
-EOF
-
-mkdir -p /opt/adb
-base64 -d < cert-gen.sh.base64 > cert-gen.sh
-mv cert-gen.sh /opt/adb/
-chmod u+x /opt/adb/cert-gen.sh
```
Comment (navidshaikh) [inline build_tools/kickstarts/centos-7-adb-vagrant.ks:190]: Will the script be `u+x` in adb-utils?

Comment (praveenkumar) [inline build_tools/kickstarts/centos-7-adb-vagrant.ks:190]: yes it is have `+x` flag which cover [u]ser, [g]roup, [o]thers.  Installed package make sure below permission for users hope that is expected.

```
ls -l /opt/adb/
total 4
-rwxr-xr-x. 1 root root 2882 Feb  9 07:30 cert-gen.sh
```

Comment (LalatenduMohanty) [inline build_tools/kickstarts/centos-7-adb-vagrant.ks:190]: @praveenkumar this is interesting. We have not put anything specific for the permissions in the adb-utils spec file. Wondering how it got +x?

Comment (praveenkumar) [inline build_tools/kickstarts/centos-7-adb-vagrant.ks:190]: @LalatenduMohanty  it's part of source file permission, not part of spec file.

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**